src/components/Homepage/HomeExp/KivaMultiCategoryGrid.vue

```diff
@@ -1,35 +1,48 @@
 <template>
-	<div class="md:tw-flex md:tw-items-start">
-		<div class="tw-mr-4">
-			<p class="tw-text-h3 tw-font-medium tw-mb-2 tw-text-secondary">
-				I want to support
-			</p>
-			<loan-category-selector-home-exp
+	<kv-grid class="tw-grid-cols-12">
+		<div class="tw-col-span-12 md:tw-col-span-3">
+			<div class="md:tw-items-start">
```

Comment:
Since this doesn't have `tw-flex` I don't think the `md:tw-items-start` class is doing anything. This wrapper div can probably be removed.